manifest.md

```diff
@@ -144,28 +138,33 @@ Unlike the [image index](image-index.md), which contains information about a set
 }
 ```
 
-## Example of a SCRATCH config or layer descriptor
+## Guidance for a filler descriptor
+
+*Implementers note*: The following is considered GUIDANCE for portability.
 
-Notice that the `mediaType` is subject to the usage or context, while the digest is specifically defined as `ScratchDigestSHA256`.
-When the `ScratchDigestSHA256` is used, the media type SHOULD be set to `application/vnd.oci.scratch.v1+json` to differentiate the descriptor from one pointing to content.
+Parts of the spec necessitate including a descriptor to a blob where some implementations of artifacts do not have associated content.
+While an empty blob (`size` of 0) may be preferable, practice has shown that not to be ubiquitously supported.
+The media type `application/vnd.oci.filler.v1+json` (`MediaTypeFiller`) has been specified for a descriptor that has no content for the implementation.
+The blob payload is the most minimal content that is still valid JSON object: `{}` (`size` of 2).
+The blob digest of `{}` is `sha256:44136fa355b3678a1146ad16f7e8649e94fb4fc21fe77e8310c060f61caaff8a`.
 
-```json,title=SCRATCH%20config&mediatype=application/vnd.oci.descriptor.v1%2Bjson
+The resulting descriptor shown here is also defined in reference code as `FillerDescriptor`:
+
+```json,title=filler%20config&mediatype=application/vnd.oci.descriptor.v1%2Bjson
 {
-  "mediaType": "application/vnd.oci.scratch.v1+json",
+  "mediaType": "application/vnd.oci.filler.v1+json",
   "size": 2,
   "digest": "sha256:44136fa355b3678a1146ad16f7e8649e94fb4fc21fe77e8310c060f61caaff8a"
```

Comment:
It might be a little "extra", but this seems like a nice place IMO for us to encourage use of the `data` field (#826) too: :innocent:

```suggestion
  "digest": "sha256:44136fa355b3678a1146ad16f7e8649e94fb4fc21fe77e8310c060f61caaff8a",
  "data": "e30="
```

Reply:
I started to do that, and worried that people might not realize that it is base64 encoded `{}`. That can probably be mitigated with another line in the description above.